src/analyze/analyze.c

```diff
@@ -2145,7 +2149,22 @@ static int do_condition(int argc, char *argv[], void *userdata) {
 }
 
 static int do_verify(int argc, char *argv[], void *userdata) {
-        return verify_units(strv_skip(argv, 1), arg_scope, arg_man, arg_generators);
+        int r;
+        char **filenames;
+
+        if (arg_alias) {
+                if (argc > 2)
+                        return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
+                                               "Only one unit can be passed with --alias");
+                filenames = &arg_alias;
```

Comment:
ahum. the list of strings must be NULL terminated themselves. This here doesn#t work, since there is no NULL pointer following the pointer to arg_alias